From 2ce224c4f4ed64b4f2e4d1cc1b0c7fe61d18828c Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:23:06 -0400
Subject: [redhat] kabi: Adding symbol down_read_trylock (kernel/rwsem.c)

Message-id: <b32172b902871b2083b61e4e2c568831f7904d37.1410510773.git.poros@redhat.com>
Patchwork-id: 95954
O-Subject: [PATCH RHEL7.1 066/233] kabi: Adding symbol down_read_trylock (kernel/rwsem.c)
Bugzilla: 1071098
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Used to acquire semaphore for reading without waiting.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/down_read_trylock
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/down_read_trylock
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/down_read_trylock

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/down_read_trylock b/redhat/kabi/kabi-module/kabi_ppc64/down_read_trylock
new file mode 100644
index 0000000..2a5297b
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/down_read_trylock
@@ -0,0 +1,2 @@
+#1-
+0xc91ff664 down_read_trylock vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/down_read_trylock b/redhat/kabi/kabi-module/kabi_s390x/down_read_trylock
new file mode 100644
index 0000000..a5038a5
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/down_read_trylock
@@ -0,0 +1,2 @@
+#1-
+0x75d22842 down_read_trylock vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/down_read_trylock b/redhat/kabi/kabi-module/kabi_x86_64/down_read_trylock
new file mode 100644
index 0000000..3db96fb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/down_read_trylock
@@ -0,0 +1,2 @@
+#1-
+0xa3a77c69 down_read_trylock vmlinux EXPORT_SYMBOL
-- 
1.7.1